### DISCUSSION TITLE
Export and Reporting Functionality

### DISCUSSION THREAD

**Discussion by arpit-982** [ananthakumaran/paisa#216] [Ideas]
Would love to see some way to export data out of the app (with ability to export out custom reports or transactions) in a .csv format as it can be useful for sending it to other apps or generally using your financial data in a spreadsheet. Seeing as ledger natively offers a robust set of tools for reporting and exporting said reports. Would love to see some functionality come in here as the import using .csv is already pretty good.

**Comment by arpit-982** [comment]
Great to see support for exporting transactions. But its not respecting the search/filter functionality in the transaction page. Clicking on download always downloads all transactions present in the ledger even when the count near it reduces as per filters/search applied. As you can always download the whole set of transactions if you don't put in any search filters, after searching you should be able to just download the filtered results as per the search query. 

<img width="1735" alt="image" src="https://github.com/user-attachments/assets/b6182cc1-def0-471d-9fc5-68253e78b663" />

**Comment by SagnikPradhan** [comment]
@ananthakumaran It looks like this is still not working. Do you think it could be prioritised soon?
